### PR TITLE
fix(TDP-4496): remove modal enter key management

### PR DESCRIPTION
**Pull request by frassinier**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4496

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [x] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1/)

File: dataprep-webapp/src/app/components/widgets/modal/widget-modal-directive.spec.js [hunk 1]
```diff
@@ -463,10 +424,10 @@ describe('Modal directive', () => {
 			//when
 			scope.state = true;
 			scope.$digest();
-			$timeout.flush();
 
 			//then
 			const activeElement = document.activeElement; //eslint-disable-line angular/document-service
+			console.log(activeElement);
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/widgets/modal/widget-modal-directive.spec.js]: FIXME

File: dataprep-webapp/src/app/components/widgets/modal/widget-modal-directive.js [hunk 2]
```diff
@@ -263,14 +253,21 @@ export default function TalendModal($timeout) {
...
 							innerElement.focus();
 
-							$timeout(() => {
-								// focus on first input (ignore first because it's the state checkbox)
-								const inputs = iElement.find('input:not(".no-focus")').eq(1);
-								if (inputs.length) {
-									inputs.focus();
-									inputs.select();
+							// focus on first input (ignore first because it's the state checkbox)
+							const inputs = innerElement.find('[type="checkbox"], [type="email"], [type="search"], [type="submit"], [type="radio"], [type="text"], select, textarea');
```
Comment (ncomont) [inline dataprep-webapp/src/app/components/widgets/modal/widget-modal-directive.js]: Could `innerElement.find('input')` do the job ?

File: dataprep-webapp/src/app/components/widgets/modal/widget-modal-directive.spec.js (hunk 1 above)
Comment (ncomont) [inline dataprep-webapp/src/app/components/widgets/modal/widget-modal-directive.spec.js]: Could you remove this, please ?

File: dataprep-webapp/src/app/settings/actions/dataset-actions-service.js [hunk 3]
```diff
@@ -88,7 +88,7 @@ export default class DatasetActionsService {
 			const dataset = action.payload.model;
 			this.TalendConfirmService
 				.confirm(
-					{ disableEnter: true },
+					{},
```
Comment (ncomont) [inline dataprep-webapp/src/app/settings/actions/dataset-actions-service.js]: Does this parameter really usefull ? It's always set to `{}` or `null` accross the app.

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/2/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/2/)

File: dataprep-webapp/src/app/settings/actions/dataset-actions-service.js (hunk 3 above)
Comment (frassinier) [reply dataprep-webapp/src/app/settings/actions/dataset-actions-service.js]: You right, will remove it.

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/5/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/5/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/6/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/6/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/8/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/8/)

**Pull request merged**